internal/core/command/restDevice.go

```diff
@@ -132,7 +128,7 @@ func restGetAllCommands(w http.ResponseWriter, _ *http.Request) {
 	status, devices, err := getCommands()
 	if err != nil {
 		LoggingClient.Error(err.Error(), "")
-		w.WriteHeader(http.StatusInternalServerError)
+		w.WriteHeader(status)
 	} else if status != http.StatusOK {
```

Comment:
This else can be removed, and just do something like this:
```go
if err != nil {
	LoggingClient.Error(err.Error())
}
w.WriteHeader(status)
```

Reply:
I address this in PR #933 

We actually don't want to fall through in the case where `status != http.StatusOK` because we don't want to set a content type.